### PR TITLE
fix(storage): remove disks with a depth higher than /dev

### PR DESCRIPTION
**Pull request by rgallor**
This should temporarily provide a solution to the second point explained in #232

### CODE REVIEW

File: src/telemetry/storage_usage.rs [hunk 1]
```diff
@@ -33,19 +33,35 @@ pub struct DiskUsage {
...
-            DiskUsage {
-                totalBytes: disk.total_space() as i64,
-                freeBytes: disk.available_space() as i64,
+    let res = sys
+        .disks()
+        .iter()
+        .filter(|disk| disk.name().to_str().is_some())
+        .map(|disk| {
+            (
+                disk.name().to_str().unwrap().replace("/dev/", ""),
```
Comment (harlem88) [inline src/telemetry/storage_usage.rs:45]: Remove the `unwrap()`

**Review by harlem88 — CHANGES_REQUESTED** [review]

File: src/telemetry/storage_usage.rs [hunk 2]
```diff
@@ -33,19 +33,35 @@ pub struct DiskUsage {
...
+                disk.total_space(),
+                disk.available_space(),
+            )
+        })
```
Comment (joshuachp) [inline src/telemetry/storage_usage.rs:49]: This can be simplified in a `flat_map`

File: src/telemetry/storage_usage.rs [hunk 3]
```diff
@@ -33,19 +33,35 @@ pub struct DiskUsage {
...
+            |mut ret, (name, total_space, free_space)| {
+                ret.insert(
+                    name.to_owned() as String,
+                    DiskUsage {
+                        totalBytes: total_space as i64,
+                        freeBytes: free_space as i64,
+                    },
+                );
+                ret
             },
```
Comment (joshuachp) [inline src/telemetry/storage_usage.rs:63]: Prefer collect to fold

File: src/telemetry/storage_usage.rs [hunk 4]
```diff
@@ -33,19 +33,35 @@ pub struct DiskUsage {
...
-    for disk in sys.disks() {
-        let disk_name = disk.name().to_str().unwrap().replace("/dev/", "");
-        ret.insert(
-            disk_name.to_owned() as String,
-            DiskUsage {
-                totalBytes: disk.total_space() as i64,
-                freeBytes: disk.available_space() as i64,
+    let res = sys
+        .disks()
+        .iter()
```
Comment (joshuachp) [inline src/telemetry/storage_usage.rs:41]: Can this be a `into_iter()`? To have an owned value

**Review by joshuachp — CHANGES_REQUESTED** [review]

**Review by harlem88 — APPROVED** [review]

**Review by joshuachp — APPROVED** [review]

**Pull request merged**